xml/cl.xml

```diff
@@ -1899,7 +1899,9 @@ server's OpenCL/api-docs repository.
         <enum value="0x2067"        name="CL_EXTERNAL_MEMORY_HANDLE_DMA_BUF_KHR"/>
         <enum value="0x2068"        name="CL_SEMAPHORE_HANDLE_OPAQUE_WIN32_NAME_KHR"/>
         <enum value="0x2069"        name="CL_EXTERNAL_MEMORY_HANDLE_OPAQUE_WIN32_NAME_KHR"/>
-            <unused start="0x206A" end="0x2FFF" comment="Reserved to Khronos for interop"/>
+            <unused start="0x206A" end="0x206F" comment="Reserved to Khronos for interop"/>
+        <enum value="0x2070"        name="CL_EXTERNAL_MEMORY_HANDLE_ANDROID_HARDWARE_BUFFER_KHR"/>
```

Comment:
Minor: Could this enum use value `0x206A`, or is it too late to change this?

Reply:
Discussed in the May 6th teleconference.  We will keep the existing token assignment.